cmd/tdex/depositfee.go

```diff
@@ -2,16 +2,38 @@ package main
 
 import (
 	"context"
+	log "github.com/sirupsen/logrus"
+	"github.com/tdex-network/tdex-daemon/pkg/explorer"
+	"github.com/tdex-network/tdex-daemon/pkg/trade"
+	"github.com/tdex-network/tdex-daemon/pkg/wallet"
+	"time"
 
 	pboperator "github.com/tdex-network/tdex-protobuf/generated/go/operator"
 
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	MinFee          = 5000
+	MaxNumOfOutputs = 150
+)
+
 var depositfee = cli.Command{
 	Name:   "depositfee",
 	Usage:  "get a deposit address for the fee account used to subsidize liquid network fees",
 	Action: depositFeeAction,
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "no-fragment",
```

Comment:
I was thinking that maybe better do the opposite, by default not doing anything.

Would modify this as just `--fragment`